### ISSUE TITLE
Webpack target node doesn't emits the built in require function

### ISSUE THREAD

**Issue by ArielGueta** [webpack/webpack#11758]
Based on the official docs, if we set the target to node, Webpack will compile for usage in a Node.js-like environment (uses Node.js require to load chunks and not touch any built-in modules like fs or path).

For example:
```js
module.exports = {
  entry: './src/one.js',
  target: 'node',
  mode: 'development',
  output: {
    filename: 'bundle.js',
    path: path.resolve(__dirname, 'dist'),
  },
};
```

But when I run the code, I still see in the output the `__webpack_require__` function, and not the original `require` function built-in node. When targeting `node`, I'm expecting Webpack to use the built-in `require` function.

**Comment by evilebottnawi** [comment]
Thank you for creating this issue. However, issues need to follow one of our templates so that we can clearly understand your particular circumstances.

**Comment by ArielGueta** [comment]
Ok, I'll update it soon, don't close it, please.

**Comment by ArielGueta** [comment]
Done

**Comment by evilebottnawi** [comment]
@ArielGueta No problems:
```
module.exports = require(\"fs\");
```

**Comment by evilebottnawi** [comment]
We have `fs` wrapper, but it expected

**Comment by ArielGueta** [comment]
What do you mean? Why do we need to use Webpack custom require function when targeting node?

**Comment by evilebottnawi** [comment]
Because that's how it works

**Comment by ArielGueta** [comment]
But the docs say this:

> uses Node.js require to load chunks

**Comment by evilebottnawi** [comment]
What is the problem? Yes webpack generates this code because it is require right now

**Comment by ArielGueta** [comment]
"uses Node.js require to load chunks" doesn't mean "uses Node.js `require()` function to load chunks"?

**Comment by evilebottnawi** [comment]
We use `require` to load node modules

**Comment by ArielGueta** [comment]
No, you're using __webpack_require__ to load them. See the output code in the issue.

**Comment by evilebottnawi** [comment]
We need to fix docs
